src/test/kotlin/com/petqua/test/fixture/WishFixtures.kt

```diff
@@ -0,0 +1,15 @@
+package com.petqua.test.fixture
+
+import com.petqua.domain.wish.Wish
+
+fun wish(
+    id: Long = 0,
+    productId: Long = 1,
+    memberId: Long = 1,
```

Comment:
```suggestion
    id: Long = 0L,
    productId: Long = 1L,
    memberId: Long = 1L,
```
통일 하시죠..!
kotlin은 L 없어도 컴파일이 되긴 되네요!(이게 정상이라고 생각합니다)

Reply:
좋습니다!!

5add123e229fe9ed00e8ea187e2e7d0f057261c9